Makefile

```diff
@@ -171,6 +172,8 @@ Q := $(if $V,,@)
 
 $(gobin_tool):
 	GO111MODULE=off go get -u github.com/myitcv/gobin
+	# NOTE: Add the GOPATH to the PATH to find gobin
+	export PATH=$(shell go env GOPATH)/bin:$(PATH)
```

Comment:
This is not full-proof as `GOPATH` can be a list.
Probably it's best to leave this to the caller to make sure this is setup.

Reply:
I didn't like this change, I removed. The problem comes from 
`gobin_tool ?= $(shell which gobin || echo $(GOPATH)/gobin)` which should be `gobin_tool ?= $(shell which gobin || echo $(GOPATH)/bin/gobin)` with `$(GOPATH)/bin/gobin`. That way it'll find the gobin path. Also, I believe we should change the references to gobin in the Makefile to then use `$(gobin_tool)`. Do you like that solution ?

Reply:
@hectorj2f yeah, not using $(gobin_tool) was a mistake on my part.